### DISCUSSION TITLE
can Isaar be used as a backend database ?

### DISCUSSION THREAD

**Discussion by dadja** [isar/isar#1176] [Help]
hello here i hope you all doing great . i was wondering if i can use Isar with dart_frog toi build a backend for a Saas project instead of the common  sql or postegreSql. can it hanble like mysql multiple connection to the database ?

**Comment by TristanGrobler** [comment]
From my many hours trying to get ISAR to work with dart_frog my results are as follows:

Implementing it and running using dart_frog dev works great.

However, once using dart_frog build and then building the image and running it, the image crashes on startup.

Some ISAR lib error. Still trying to find a solution to this. . .
